.github/workflows/build.yml

```diff
@@ -0,0 +1,101 @@
+# /********************************************************************************
+# * Copyright (c) 2022 Contributors to the Eclipse Foundation
+# *
+# * See the NOTICE file(s) distributed with this work for additional
+# * information regarding copyright ownership.
+# *
+# * This program and the accompanying materials are made available under the
+# * terms of the Apache License 2.0 which is available at
+# * https://www.apache.org/licenses/LICENSE-2.0
+# *
+# * SPDX-License-Identifier: Apache-2.0
+# ********************************************************************************/
+
+name: BitBake Build
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  leda-qemux86:
```

Comment:
qemux86 or  qemux86 as it will be in  KAS_MACHINE=

Reply:
Agree, will change job names to match machine ids.